Documentation/user-guides/getting-started.md

```diff
@@ -11,63 +11,35 @@ draft: false
 description: Getting started guide
 ---
 
-The Prometheus Operator's goal is to make running Prometheus on top of Kubernetes
-as easy as possible, while preserving Kubernetes-native configuration options.
-
-This guide will show you how to deploy the Prometheus operator, set up a
-Prometheus instance, and configure metrics collection for a sample application.
+The Prometheus Operator introduces custom resources in Kubernetes to declare
+the desired state of a Prometheus and Alertmanager cluster as well as the
```

Comment:
not sure why we talk about Alertmanager here since the guide seems focused on Prometheus + monitor resources. Maybe add a note that the operator can also manage other custom resources but not mention them?

Reply:
> Should we move this to the top? I like that a getting started guide states what it covers upfront.

If we move the statement that we will cover ServiceMonitor and PodMonitor in this guide to the top, there is no need to discuss this point. The next line again talks about Service and PodMonitor so this feels like it's not needed in between the two points. Wdyt?